bleu/op-bacchus
# Jean/op 231 integrate emit creat event attestion when submitting new

Now following the create event form will call the attestation

How to test:
  Connect wallet
  Go to /events/new
  Fill the overview form
  Fill the description form
  Accept the transaction
  Go to /events/
  check the new attestation is there

Also refactored app header (removed admin buttons and fixed Events dropdown)

**Note**: we just discovered there are better ways to implement a multistep form, such as [Wizard](https://vee-validate.logaretm.com/v4/examples/multistep-form-wizard/)
